src/DataCollectors/Microsoft.TestPlatform.Extensions.EventLogCollector/EventLogDataCollector.cs

```diff
@@ -383,7 +383,7 @@ private void RemoveTempEventLogDirs(List<string> tempDirs)
         }
     }
 
-    private void StartCollectionForContext(DataCollectionContext dataCollectionContext, bool isSessionContext)
```

Comment:
Looks like there was some intent behind having this parameter, did someone from our current team write it? Can you confirm with them it is unused on purpose please?